File: Py/calculator.py
operation = ''
while operation != "done":

    print( "\n add \n subtract \n multiply \n divide \n")
    operation = input(': ')
    if operation != 'done':
        first = int(input('first number?'))
        last = int(input('last number?'))

    if operation == "add":
        addition = input(first + last)
        print(addition)

    elif operation == "multiply":
        multiply = input(int(first) * int(last))
        print(multiply)

    elif operation == "subtract":
        subtraction = input(int(first) - int(last))
        print(subtraction)

    elif operation == "divide":
        division = input(int(first) / int(last))
        print(divide)

File: Js/calculator.js

let operation = {}
  let one, two, op, answer

while (operation != "done")
{
  one = prompt("first number")
  two = prompt("second number")
  operation = prompt("add/sub/mult/div?")

  if (operation == "add"){
    answer = one + two
  }

  if (operation == "sub"){
    answer = one - two
  }

  if (operation == "mult"){
    answer = one * two
  }

  if (operation == "div"){
    answer = one / two
  }

  if (answer != undefined){
    operation = "done"
  }
}
console.log(answer)
